cmp/grid/GridModel.ts

```diff
@@ -1771,7 +1771,11 @@ export class GridModel extends HoistModel {
     }
 
     defaultGroupSortFn = (a, b) => {
-        return a < b ? -1 : a > b ? 1 : 0;
+        // Place ungrouped items at bottom.
+        if (a == b) return 0;
```

Comment:
Maybe more appropriate to use strict equality checks?  (i.e. `(0 == '') = true` but `(0 === '') = false`).

Reply:
fixed